backend/CircleForms/Services/Database/PostRepository.cs

```diff
@@ -181,6 +181,28 @@ public async Task AddAnswer(string postId, Answer entry)
         }
     }
 
+    public async Task<bool> AddPinnedPosts(string postId)
+    {
+        var redisDb = _redis.GetDatabase();
+        var key = $"post:{postId}";
+
+        if (!redisDb.KeyExists(key))
+        {
+            return false;
+        }
+
+        await redisDb.SetAddAsync("posts:pinned", key);
```

Comment:
Do we not want to store pinned-ness in the mongo as well?

Reply:
i dont think so, probably we can add 
`public bool IsPinned { get; set; }` 
property, but it seems quite redundant atm

Reply:
later on it should be done, but not right now

Reply:
sure